docs/guides/setup-channel.mdx

```diff
@@ -49,55 +48,47 @@ Available plans:
-    * If you don't save your credentials, you must specify them every time you start a new session.
-    * The `channel` parameter allows you to distinguish between different account types. When initializing the account, IBM Cloud is the default account used if have saved credentials for an IBM Quantum Platform and an IBM Cloud account.
+      * If you have saved credentials for both an IBM Quantum Platform account and an IBM Cloud account, IBM Cloud is the default account used when you initialize your account, unless you specify `set_as_default=True` in your IBM Quantum Platform account when you use the `save_account()` method.
+      * If you are saving multiple accounts per channel, consider using the `name` parameter to differentiate them.
+      * Credentials are saved to `$HOME/.qiskit/qiskit-ibm.json`.  Do not manually edit this file.
 
-    <Admonition type="caution">
-      Account credentials are saved in plain text, so only do so if you are using a trusted device.
+    <Admonition type="warning">
+      **Protect your API token!** Never include your token in plain text in a notebook file. If you are working in a trusted Python environment (such as on a personal laptop or workstation), use the `save_account()` method described in the previous step to store your credentials, keeping in mind that your token will still be stored as plain text on your local drive. If you are executing code in a Python environment that is not trusted, see the [instructions in the next step](#untrusted-machine).
     </Admonition>
 
-    <Admonition type="note">
-    IBM Cloud is the default account used if you don't specify a different channel or account name.
-    </Admonition>
+1. <span id="untrusted-machine"></span>**Avoid executing code on an external or random cloud Python environment to minimize security risks.** If you must use an untrusted environment (on, for example, a public computer), change your API token after each use by rotating it on the [IBM Quantum Platform dashboard](https://quantum.ibm.com/) (click the refresh button in the API token field). To initialize the service in this situation, you can use code like the following:
 
-    <CodeAssistantAdmonition
-      tagLine="If you forget the setup code, try asking Qiskit Code Assistant."
-      prompts={[
-        "# Set up my IBM Runtime account using TOKEN for my token"
-      ]}
-    />
+    ```python
+    from qiskit_ibm_runtime import QiskitRuntimeService
+  
+    # Refresh your API token on the dashboard after using the code as follows:
+    service = QiskitRuntimeService(channel="ibm_quantum", token="<MY_IBM_QUANTUM_TOKEN>")
+    ```
+
+    <Admonition type="warning">
+      When sharing code with others, ensure that your API token is not embedded directly within the Python script in plain text. Instead, share the script without the token and provide instructions for securely setting it up.
```

Comment:
```suggestion
      When sharing code with others, ensure that your API token is not embedded directly within the Python script. Instead, share the script without the token and provide instructions for securely setting it up.
```

Shouldn't include the token at all.

Reply:
I'd also add `.. Python script or notebook`... since notebooks are the most common way for people to share.

Reply:
resolved by 1f3d40de4d4fdcde7f47199b13ace582c4466670 and 0e18cd37b89489d3fcfb0d38a41d5163e9134178